src/Azure.Functions.Cli/Common/Constants.cs

```diff
@@ -49,12 +49,13 @@ public static class ArmConstants
 
         public static class DockerImages
         {
-            public const string LinuxPythonImageAmd64 = "mcr.microsoft.com/azure-functions/python:2.0";
+            public const string LinuxPythonImageAmd64 = "mcr.microsoft.com/azure-functions/python:2.0.12134";
```

Comment:
Leave this `2.0`, I'll make sure to update that tag with `2.0.12134` before the CLI release. It's fine to keep it with this specific tag on your machine for testing, but the final commit should be `2.0`

Reply:
That sounds good.